sdk/examples/gateway.ts

```diff
@@ -51,13 +49,8 @@ async function createTxWithOpReturn(fromAddress: string, toAddress: string, amou
     );
 
     const psbt = unsignedTx.toPSBT(0);
-    const psbtHex = hex.encode(psbt);
 
-    // TODO: sign PSBT
-    const signedPsbtHex = psbtHex;
+    const signedTx = Transaction.fromPSBT(psbt);
 
-    const signedTx = SigTx.fromPSBT(bitcoin.Psbt.fromHex(signedPsbtHex).toBuffer());
-    signedTx.finalize();
-
-    return bitcoin.Transaction.fromBuffer(Buffer.from(signedTx.extract()));
+    return Buffer.from(signedTx.extract())
```

Comment:
Wondering if it makes more sense to return a string?

Reply:
`Buffer.from(signedTx.extract()).toString('base64')` is this what you expect?

Reply:
We should return what the wallets expect to sign. I think that would be a Base64 encoded buffer. but worthwhile checking what sats-connect, xverse, unisat etc want.